CODEOWNERS

```
@@ -0,0 +1 @@
+@VantaInc/grc-partner-enabled-experiences
```

Do we need to specify the files the team owns?

```suggestion
* @VantaInc/grc-partner-enabled-experiences
```

---

Yes, bad copy paste on my part